lib/support/util.js

```diff
@@ -14,5 +16,45 @@ module.exports = {
           formatted += ` (±${fmt(metric.mdev)})`;
       }
       return formatted;
+  },
+  getResultLogLine(result, options) {
+    // don't bother if no statistics or silent x2
+    if (!result.statistics || !result.statistics.timings || !result.statistics.timings.pageTimings || options.silent > 1) return result;
```

Comment:
Since we now have data to output even if there's just har data (and no statistics) I don't think this check is relevant. Even if it is, it should return an empty string.